spec/defines/004_elasticsearch_plugin_spec.rb

```diff
@@ -294,7 +294,13 @@ class { 'elasticsearch': }
           'Class[elasticsearch::config]'
         )}
 
-        include_examples('class')
+        it { should contain_elasticsearch_plugin(
+          'head'
+        ).that_comes_before(
```

Comment:
This can be chained into the previous `it` block...

So:
```puppet
        ).that_requires(
          'Class[elasticsearch::config]'
        ).that_comes_before(
          'Service[elasticsearch]'
        )}
```

Reply:
It can't, those are 2 different resources. `contain_elasticsearch__plugin != contain_elasticsearch_plugin`

Reply:
Doh, you're right... Good :eye: :)